GraphicEditor/View/Building1FirstFloorPlan.xaml.cs

```diff
@@ -0,0 +1,219 @@
+                    rectangle.MouseDown += AdditionalInformationAuxiliaryRoom;
+                    rectangle.MouseEnter += AuxiliaryRoom_MouseEnter;
+                    rectangle.MouseLeave += AuxiliaryRoom_MouseLeave;
+                }
+            }
+
+        }
+        private void mouseClickArrowUp(object sender, MouseButtonEventArgs e)
+        {
+            page.MainFrame.Content = new Building1SecondFloorPlan();
+            if (page.tabControl.SelectedIndex == 0)
+            {
+                page.comboBoxH1.SelectedIndex = 2;
+            }
+            else
+            {
+                page.comboBoxHospital1.SelectedIndex = 2;
+            }
+        }
+        private void mouseClickArrowDown(object sender, MouseButtonEventArgs e)
+        {
+            page.MainFrame.Content = new Building1FloorPlan();
+            if (page.tabControl.SelectedIndex == 0)
+            {
+                page.comboBoxH1.SelectedIndex = 0;
+            }
+            else
+            {
+                page.comboBoxHospital1.SelectedIndex = 0;
+            }
+        }
+
+        private void PatientRoom_MouseEnter(object sender, MouseEventArgs e)
+        {
+            if (ChooseUser.Restriction == 0)
+            {
+                PopupPatientRoom.Placement = PlacementMode.MousePoint;
+                PopupPatientRoom.IsOpen = true;
+            }
+            else
+            {
+                PopupInfoForPatient.Placement = PlacementMode.MousePoint;
+                PopupInfoForPatient.IsOpen = true;
+            }
+        }
+
+        private void PatientRoom_MouseLeave(object sender, MouseEventArgs e)
+        {
+            PopupPatientRoom.Visibility = Visibility.Collapsed;
+            PopupPatientRoom.IsOpen = false;
+
+            PopupInfoForPatient.Visibility = Visibility.Collapsed;
+            PopupInfoForPatient.IsOpen = false;
+        }
+
+        private void ExaminationRoom_MouseEnter(object sender, MouseEventArgs e)
+        {
+            PopupExaminationRoom.Placement = PlacementMode.MousePoint;
+            PopupExaminationRoom.IsOpen = true;
+        }
+
+        private void ExaminationRoom_MouseLeave(object sender, MouseEventArgs e)
+        {
+            PopupExaminationRoom.Visibility = Visibility.Collapsed;
+            PopupExaminationRoom.IsOpen = false;
+        }
+
+        private void OperatingRoom_MouseEnter(object sender, MouseEventArgs e)
+        {
+            if (ChooseUser.Restriction == 0)
+            {
+                PopupOperatingRoom.Placement = PlacementMode.MousePoint;
+                PopupOperatingRoom.IsOpen = true;
+            }
+            else
+            {
+                PopupInfoForPatient.Placement = PlacementMode.MousePoint;
+                PopupInfoForPatient.IsOpen = true;
+            }
+        }
+
+        private void OperatingRoom_MouseLeave(object sender, MouseEventArgs e)
+        {
+            PopupOperatingRoom.Visibility = Visibility.Collapsed;
```

Comment:
Same as previous comment,I suggest you to apply multiple assignments everywhere this situation appears